tests/device_tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+import pytest
+
+pytest.register_assert_rewrite("tests.common")
```

Comment:
could we just do this in `conftest.py` ?

Reply:
Docs say we could, but I saw we have the same thing in `tests/click_tests/__init__.py`, so I put it into `__init__` here as well

Reply:
yeah but click_tests register a file in the click_tests subfolder, i.e., "their own"

Reply:
Sure, so I will put it into `conftest.py`, as it is common for all tests - literally

Reply:
Moved in d2cee34